emscripten.py

```diff
@@ -1881,7 +1899,12 @@ def debug_copy(src, dst):
     ])
     debug_copy(base_wasm, 'base_wasm.wasm')
 
-    shared.check_call([wasm_emscripten_finalize, base_wasm, '-o', wasm])
+    shared.check_call([
+        wasm_emscripten_finalize,
+        '--emscripten-reserved-function-pointers=%d' %
+        shared.Settings.RESERVED_FUNCTION_POINTERS,
```

Comment:
I'd indent this here, the alignment looks misleading (i.e. it looks like two separate arguments rather than modifying the string). Might be worth adding parens around this to make it even clearer (also python's indenting rules means maybe you have to?)

Reply:
Done